AlayaCare Cloud Tech Radar.csv

```diff
@@ -9,35 +9,39 @@
 "Adopt","Jest","languages-and-frameworks","FALSE","<p><a href='https://jestjs.io/'>Jest</a> is a JavaScript Testing Framework with a focus on simplicity.</p>"
 "Adopt","Kubernetes","languages-and-frameworks","FALSE","<p><a href='https://kubernetes.io/'>kubernetes.io</a></p>"
 "Adopt","Liquibase","Tools","FALSE","<p><a href='https://www.liquibase.org/'>liquibase</a></p>"
-"Adopt","MariaDB","Platforms","FALSE","<p></p>"
+"Adopt","CircleCI","Platforms","FALSE","<p><a href='https://circleci.com/'>circleci.com</a></p>"
 "Adopt","PostGres","Platforms","FALSE","<p></p>"
+"Adopt","MySQL","Platforms","FALSE","<p></p>"
+"Adopt","Microsoft Visio","Tools","TRUE","<p>Recommended for individual work when then end result requires polishing, e.g. C4 architecture diagrams.</p>"
```

Comment:
> C4 architecture diagrams

how do you do C4 with visio ? there is a no predefined shapes, no template we can use out of the box (with draw.io, there are predefined C4 shapes you can use right out of the box, which makes the C4 diagrams consistent in look and feel for anyone who builds one)

so, really, "when then end result requires polishing" I think is a wrong use case...